packages/puppeteer-core/src/common/bidi/Page.ts

```diff
@@ -557,6 +558,228 @@ export class Page extends PageBase {
     }
   }
 
+  #exposedFunctionChannels = (function* () {
+    let id = Math.random();
```

Comment:
why Math.random and not 0?

Reply:
In case there are other clients connected to the server. The channel names are global to the process, so we can't start from 0.